project/AppDependencies.scala

```diff
@@ -7,8 +7,8 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-play-26" % "1.7.0",
-    "uk.gov.hmrc" %% "domain" % "5.8.0-play-26"
+    "uk.gov.hmrc" %% "bootstrap-play-26" % "1.13.0",
+    "uk.gov.hmrc" %% "domain" % "5.9.0-play-26"
```

Comment:
Do you know if we actually need domain? I heard someone mention that when we upgrade to Play 2.7 this needs to go because there's no support for it.

Reply:
I've deleted domain and service works fine and tests pass